oar/cli/cmd_image_consistency_check.py

```diff
@@ -58,18 +74,21 @@ def image_consistency_check(ctx, build_number):
                     LABEL_TASK_IMAGE_CONSISTENCY_TEST, TASK_STATUS_INPROGRESS
                 )
     else:
-        logger.info(
-            f"check image-consistency-check job status with job id:{build_number}"
-        )
-        job_status = jh.get_build_status(
-            "image-consistency-check", build_number)
+        if(for_nightly):
+            logger.error(f"no need to add '-c' option,if just want to check job status")
```

Comment:
@wewang58 error message needs to be updated accordingly

Reply:
@rioliu-rh  Yes, updated